airflow/hooks/presto_hook.py

```diff
@@ -44,18 +45,22 @@ class PrestoHook(DbApiHook):
     def get_conn(self):
         """Returns a connection object"""
         db = self.get_connection(self.presto_conn_id)
-        reqkwargs = None
+        auth = None
         if db.password is not None:
-            reqkwargs = {'auth': HTTPBasicAuth(db.login, db.password)}
-        return presto.connect(
+            auth = prestodb.auth.BasicAuthentication(db.login, db.password)
```

Comment:
This block could be replaced with `auth = prestodb.auth.BasicAuthentication(db.login, db.password) if db.password else None`. This would also allow you to inline this at line 61.

Reply:
ok